src/main/charts/bitbucket/values.yaml

```diff
@@ -81,13 +81,16 @@ bitbucket:
     # -- The key in the Kubernetes Secret which contains the sysadmin email address
     emailAddressSecretKey: emailAddress
 
-  proxy:
+  ingress:
     # -- The fully-qualified domain name of the ingress
     fqdn:
     # -- The port number of the ingress
     port: 443
-    # -- note that, if present, the value of x-forwarded-proto header will trump this setting
+    # -- The protocol scheme used by the ingress that sends traffic to the application.
```

Comment:
Shouldn't that be "the protocol scheme used by the browser to access the application"?

Reply:
To clarify: our ingresses in EKS use HTTP, not HTTPS. HTTPS happens outside of Kubernetes.